apps/video/js/video.js

```diff
@@ -141,6 +141,12 @@ navigator.mozL10n.once(function() {
   // Tell performance monitors that our chrome is visible
   window.performance.mark('navigationLoaded');
 
+  // Reformatting eventlistener is handled by ThumbnailList
```

Comment:
actually no, I meant `ThumbnailList` because its there where the navigator.mozL10n.ready is set and it triggers `*.localize` on itself and its children classes down to this class.